.Dockerignore

```diff
@@ -0,0 +1 @@
+_site
```

Comment:
can you rename this file to be lowercase (`.dockerignore`)?

Also; looks like we no longer need Git during build, so we can now ignore `.git`, `.github`, and likely the `tests` directory (There's some other files we don't need, but excluding them likely won't improve performance);

```dockerignore
.git
.github
_site
tests
```

Reply:
Actually, this is the `docs-base` branch, so only `.git` needs to be added to `.dockerignore`. Doing so makes a big difference though:

Before:

```bash
$ docker build -t docs:base .
Sending build context to Docker daemon  541.2MB
...
```

After:

```bash
$ docker build -t docs:base .
Sending build context to Docker daemon  13.82kB
...
```